site/tool_configuration.md

```diff
@@ -1,10 +1,26 @@
 ## Tools Configuration
+ - [Tool Property File](#tool-properties)
  - [Model Filters](#model-filters)
  - [Target Environments](config/target_env.md)
  - [Type Definitions](#domain-type-definitions)
  - [Variable Injection](variable_injection.md)
  - [Custom Configuration](#custom-configuration)
 
+ ### Tool Properties
+ The WebLogic Deploy Tooling tools can be configured or tuned from the tool property file. This property file is installed as `<weblogic-deploy>/lib/tool.properties`. You may tune any of the properties in this file or override the file completely by providing your own tool.properties file. Reference the directory where the tool.properties resides using the  `CUSTOM_CONFIG_VARIABLE` Environment variable.
```

Comment:
To change this sentence to active voice, "You can configure or tune the WebLogic Deploy Tooling tools using the tool property file."
tool.properties file. -> `tool.properties` file. (code font for a file name; globally)
where the tool.properties resides -> where the `tool.properties` file resides
Environment  -> environment